src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -59,8 +60,15 @@ public void setNeutralMode() {
     rightCenteringMotor.set(0);
   }
 
+  private double getRollerPercentOutput() {
+    return rollerMotor.get();
+  }
+
   @Override
   public void periodic() {
+
     // This method will be called once per scheduler run
+    SmartDashboard.putNumber("Roller Percent Output", getRollerPercentOutput());
```

Comment:
We should have a / before denoting subsystem

Reply:
nuh uh 

https://www.gstatic.com/android/keyboard/emojikitchen/20201001/u1f431/u1f431_u1f34d.png?fbx